db/seeds.rb

```diff
@@ -26,3 +27,6 @@
 Review.create(text: "Braveheart sucks.", rating: 1, video: Video.first, user: User.first)
 Review.create(text: "Best movie ever.", rating: 4, video: Video.first, user: User.first)
 
+QueuedVideo.create(order: 1, video: Video.first, user: User.first)
```

Comment:
You should assign your created objects to variables, then you can use those variables here.